app/controllers/concerns/client_sortable.rb

```diff
@@ -81,11 +102,35 @@ def filters_from(source)
       sla_breach_date: source[:sla_breach_date],
       used_navigator: source[:used_navigator],
       ctc_client: source[:ctc_client],
+      last_contact: source[:last_contact],
+      active_returns: source[:active_returns],
     }
   end
 
+  def filtering_only_by?(filter_values)
+    @filters.select { |_k, v| v.present? } == filter_values.transform_values { |v| v.to_s }
+  end
+
   def search_and_sort_params
-    [:search, :status, :unassigned, :assigned_to_me, :flagged, :unemployment_income, :year, :vita_partners, :assigned_user_id, :language, :service_type, :greetable, :sla_breach_date, :used_navigator, :ctc_client]
+    [
+      :search,
```

Comment:
Wonder if it's possible to refactor this to share the list of keys with filtered_clients but obvi not a showstopper -al